### DISCUSSION TITLE
Invalid Notebook

### DISCUSSION THREAD

**Discussion by mmh62** [Part-II-Computational-Physics/.github#43] [General]
I keep getting invalid notebook when I try and commit any changes 

How can I go back to the original?

**Comment by JNHallen** [comment]
Hi! 

I am not sure what is causing the invalid notebook problem. You can always find any earlier version of your code by going to the repository and looking at the commits. In the command line you could do something like 
`$ git reset --hard 0ad5a7a6`
to go back to a specific version of your code.

**Comment by mmh62** [comment]
hey,

So what am I supposed to do? because I can't check if my code works or not

I have tried the 'fix' in the previous messages
________________________________

**Reply by JNHallen** [reply]
Hi! 

How are you editing the notebook? It seems like something has gone wrong in the source which means the file cannot be read as a Jupyter notebook anymore.

**Comment by mmh62** [comment]
I am now trying to do it in Github Desktop but it won't let me clone it because it says it contained files
________________________________

**Reply by JNHallen** [reply]
That is probably because you are trying to clone it to an already existing repository.
